resume-project/src/App.js

```diff
@@ -1,61 +1,84 @@
 import React from 'react';
 
+const CareerComponent = () => {
+  return(
+  <div>
+    <div className="heading">
+      <h2>
+        <i className="fas fa-user-circle"></i>Career Profile</h2>
+    </div>
+    <section className="content">
+      <h2 hidden="hidden">Career Information</h2>
+      <p>
+        After graduating from Northern Arizona University with a Bachelors
+        in Applied Computer Science I moved to Austin, Texas looking to begin
+        my career in software development. While searching for work I spent
+        a few months developing the mobile game Pazaak Cantina. I then was
+        accepted into the apprenticeship program at the software development
+        company Moove-it, located in downtown Austin. I am hopeful that I will
+        begin working as a full-time developer with Moove-it after finishing the apprenticeship.
+      </p>
+    </section>
+  </div>
+  );
+}
+
+const ExperienceComponent = (props) => {
```

Comment:
all this components should be on its own files, also do the same with the `return` statement